applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/stress_response_definitions.cpp

```diff
@@ -53,6 +53,7 @@ namespace StressResponseDefinitions
             { "MZY", TracedStressType::MZY },
             { "MZZ", TracedStressType::MZZ },
             { "PK2", TracedStressType::PK2 },
+            { "VON_MISES_STRESS", TracedStressType::VON_MISES_STRESS }
```

Comment:
@philbucher this is where it is used, if we change this we would need to adapt the others as well - definitely part of another MR if necessary.